server/sandbox.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```

Comment:
I'm happy to change it. For me tho, sandboxing is a general concept - for instance, one reason chrome uses so much memory is because it sandboxes every page from each other.